src/integration_runner/main.go

```diff
@@ -346,6 +346,14 @@ func main() {
 		ctx.Settings["newrelic.loglevel"] = *flagLoglevel
 	}
 
+        // PHP Modules common to all tests
+        ctx.Settings["zend_extension"] = "opcache.so"
```

Comment:
What happens if `php` used by integration runner is not able to load the extension?

Reply:
Good point, currently the tests just run even if it's not loaded unless a test specifically required it (like [here](https://github.com/newrelic/newrelic-php-agent/blob/ads/oapi-jit-tests/tests/integration/jit/function/skipif.inc#L12)).

We could do a few options:
1) detect and warn in log with something along the lines of "detected the default module X is missing. Tests will run, but results not guaranteed" but run tests anyway.
2) terminate with warning message "detected the default module X is missing. No tests will run." and do not run any tests.
3) A flag to toggle between option 1 and option 2

Reply:
My vote goes for 3. with the default being behavior described in 2.

Reply:
As far as I can tell, doing 2 would be quite involved. My understanding is that the integration runner is unable to know whether or not the module has been loaded until evaluating within the PHP `skipif`s. To implement this would then require an injection of the module check into the skips (which could be handled automatically), but then NOT injecting that check for tests that do not have opcache (like those added in this PR).

For simplicity's sake, I would rather avoid that altogether and just have a basic test verify that opcache was loaded. Then that singular test will fail if the extension fails to load. Then, if someone really wanted all the tests to pass without opcache, they could manually do something about that singular failure. I do not see a need to do anything more complicated.

Reply:
Added in [f7687f5](https://github.com/newrelic/newrelic-php-agent/pull/752/commits/f7687f561e5c57c30ef8cd5f23d8bc856665301a)

Reply:
go fmt?

Reply:
fixed in [4f3f31e](https://github.com/newrelic/newrelic-php-agent/pull/752/commits/4f3f31e0ab3385aa3352329e40cf17c93f03a126)